python-client/giskard/ml_worker/testing/functions/transformation.py

```diff
@@ -158,3 +158,13 @@ def mad_transformation(
         value_added = compute_mad(data[column_name])
     data[column_name] = data[column_name].apply(lambda x: x + factor * value_added)
     return data
+
+
+@transformation_function(name="ADD value", tags=["num"], row_level=False)
```

Comment:
nitpick: does "ADD" need to be capitalcase?
```suggestion
@transformation_function(name="Add value", tags=["numerical"], row_level=False)
```